Sources/PaymentButtons/PayPalButton.swift

```diff
@@ -100,7 +100,7 @@ public final class PayPalButton: PaymentButton {
         insets: NSDirectionalEdgeInsets? = nil,
         color: Color = .gold,
         edges: PaymentButtonEdges = .softEdges,
-        size: PaymentButtonSize = .collapsed,
+        size: PaymentButtonSize = .regular,
```

Comment:
We will want to update the docstring above to indicate the new default

Reply:
Good catch, thanks